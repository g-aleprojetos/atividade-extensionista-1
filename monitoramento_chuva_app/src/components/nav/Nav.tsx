import React from 'react';
import * as S from './Nav.styles';
import {TextoSmall} from 'components/texto';
import colors from 'resources/colors';
import {Menu} from 'components/menu';

export type Props = {
  handleLogout: () => void;
};

export const Nav = (props: Props) => {
  const {handleLogout} = props;
  return (
    <S.Container data-testid={'test_nav'}>
      <S.ContainerLogo>
        <S.Logo />
      </S.ContainerLogo>
      <Menu handleLogout={handleLogout} />
    </S.Container>
  );
};
