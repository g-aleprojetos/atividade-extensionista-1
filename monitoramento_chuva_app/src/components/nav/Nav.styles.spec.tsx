import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import 'jest-styled-components';
import * as S from './Nav.styles';

describe('Nav.styles', () => {
  test('Container DEVE ser igual ao snapshot', () => {
    const container = renderer.create(<S.Container />).toJSON();
    expect(container).toMatchSnapshot();
  });

  test('ContainerLogo DEVE ser igual ao snapshot', () => {
    const container = renderer.create(<S.ContainerLogo />).toJSON();
    expect(container).toMatchSnapshot();
  });
});
