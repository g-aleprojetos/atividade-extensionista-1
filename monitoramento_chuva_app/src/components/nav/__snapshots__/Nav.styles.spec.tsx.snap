// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav.styles Container DEVE ser igual ao snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  width: 100%;
}

<nav
  className="c0"
/>
`;

exports[`Nav.styles ContainerLogo DEVE ser igual ao snapshot 1`] = `
.c0 {
  position: static;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 5.563rem;
  color: #ffffff;
  padding: 0 0.5rem 0 5.563rem;
}

<div
  className="c0"
/>
`;
